std/experimental/ndslice/slice.d

```diff
@@ -62,10 +62,6 @@ auto sliced(
     if (!isStaticArray!Range && !isNarrowString!Range && N)
 in
 {
-    foreach (len; lengths)
-        assert(len > 0,
```

Comment:
Why not just make it 

```
assert(len >= 0,
```

to  allow for zero length slices while still keeping the check?